deploy/kubernetes/csi-attacher-nfsplugin.yaml

```diff
@@ -16,12 +16,15 @@ spec:
 
 ---
 kind: StatefulSet
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

since 1.16, those apps/v1beta1 and v1beta2 are deprecated